### DISCUSSION TITLE
Help: Connectors - Not Found

### DISCUSSION THREAD

**Discussion by danielnbalasoiu** [uniuuu/zotprime#5] [Q&A] [answered]
Hi,

I'm running Zotprime on a self-hosted VM, updated `config.sh` to point to VM's IP address and built Zotero client. All good so far. The problem is when I open the client a browser tab is opened (`http://VM-IP:8080/download/connector`) and I get `Not Found` error.

```
zotprime-app-zotprime-dataserver-1 | xx.yy.zz.qq - - [01/May/2023:10:38:45 +0000] "GET /download/connectors HTTP/1.1" 404 69 
```

Can you please help me debug this further?

Thanks!

**Comment by uniuuu** [comment, marked as answer]
Hi @danielnbalasoiu 
I've replied you here https://github.com/uniuuu/zotprime/issues/8
